<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorsDialog</class>
 <widget class="QDialog" name="AuthorsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Author Maintenance</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Exit Screen</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="AuthorDetails">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>341</width>
     <height>158</height>
    </rect>
   </property>
   <property name="title">
    <string>Author Details</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="DisplayLabel">
      <property name="text">
       <string>Display Name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QLineEdit" name="DisplayEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="FirstNameLabel">
      <property name="text">
       <string>First Name:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QLineEdit" name="FirstNameEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="LastNameLabel">
      <property name="text">
       <string>Last Name:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="QLineEdit" name="LastNameEdit"/>
    </item>
    <item row="3" column="0" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>198</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="DeleteButton">
      <property name="toolTip">
       <string>Delete Author</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../images/openlp-2.qrc">
        <normaloff>:/services/service_delete.png</normaloff>:/services/service_delete.png</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="AddUpdateButton">
      <property name="toolTip">
       <string>Add Update Author</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../images/openlp-2.qrc">
        <normaloff>:/system/system_settings.png</normaloff>:/system/system_settings.png</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="ClearButton">
      <property name="toolTip">
       <string>Clear Selection</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../images/openlp-2.qrc">
        <normaloff>:/songs/song_edit.png</normaloff>:/songs/song_edit.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="MessageLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="AuthorListView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>301</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="../images/openlp-2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AuthorsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AuthorsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
